pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+asyncio_mode=auto
```

Comment:
It might be nice to move this configuration into the project `pyproject.toml` file [like so](https://docs.pytest.org/en/7.3.x/reference/customize.html#pyproject-toml) to have all the project configuration is in one place.